docs/developer_guide.md

```diff
@@ -70,8 +70,9 @@ source .venv/bin/activate
 
 If no explicit action is taken, the default PyTorch version will be installed.
 This will give you a current CUDA-based version, which takes longer to download
-and includes other dependencies that SHARK does not require. To install a
-different variant, run one of these commands first:
+and includes other dependencies that SHARK does not require. In order to run eager mode inference in sharktank on a GPU, `torch+rocm` installation is required.
```

Comment:
```suggestion
and includes other dependencies that SHARK does not require. In order to run eager mode inference in sharktank on an AMD GPU, `torch+rocm` installation is required.
```
or rather with a line break, but the above for an easier review of the suggested changes
```suggestion
and includes other dependencies that SHARK does not require. In order to run eager
mode inference in sharktank on an AMD GPU, `torch+rocm` installation is required.
```